com.mousepawmedia.omission.yaml

```diff
@@ -0,0 +1,141 @@
+
+    # Dependency of Kivy-Garden (even if you don't access the network)
+    - name: python3-requests
+      buildsystem: simple
+      build-commands:
+        - pip3 install --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} requests
+      sources:
+        - type: file
+          url: https://files.pythonhosted.org/packages/19/c7/fa589626997dd07bd87d9269342ccb74b1720384a4d739a1872bd84fbe68/chardet-4.0.0-py2.py3-none-any.whl
+          sha256: f864054d66fd9118f2e67044ac8981a54775ec5b67aed0441892edb553d21da5
+        - type: file
+          url: https://files.pythonhosted.org/packages/09/c6/d3e3abe5b4f4f16cf0dfc9240ab7ce10c2baa0e268989a4e3ec19e90c84e/urllib3-1.26.4-py2.py3-none-any.whl
+          sha256: 2f4da4594db7e1e110a944bb1b551fdf4e6c136ad42e4234131391e21eb5b0df
+        - type: file
+          url: https://files.pythonhosted.org/packages/5e/a0/5f06e1e1d463903cf0c0eebeb751791119ed7a4b3737fdc9a77f1cdfb51f/certifi-2020.12.5-py2.py3-none-any.whl
+          sha256: 719a74fb9e33b9bd44cc7f3a8d94bc35e4049deebe19ba7d8e108280cfd59830
+        - type: file
+          url: https://files.pythonhosted.org/packages/a2/38/928ddce2273eaa564f6f50de919327bf3a00f091b5baba8dfa9460f3a8a8/idna-2.10-py2.py3-none-any.whl
+          sha256: b97d804b1e9b523befed77c48dacec60e6dcb0b5391d57af6a65a312a90648c0
+        - type: file
+          url: https://files.pythonhosted.org/packages/29/c1/24814557f1d22c56d50280771a17307e6bf87b70727d975fd6b2ce6b014a/requests-2.25.1-py2.py3-none-any.whl
+          sha256: c210084e36a42ae6b9219e00e48287def368a26d03a048ddad7bfee44f75871e
+
+    # We must install Kivy-Garden from sdist
+    - name: Kivy-Garden
+      buildsystem: simple
+      build-commands:
+        - sed -i 's/from ez_setup import use_setuptools//g' setup.py  # we do not need ez_setup, which accesses network
+        - sed -i 's/use_setuptools()//g' setup.py  # we do not need ez_setup, which accesses network
+        - python3 setup.py install --prefix=${FLATPAK_DEST} --root=/
+      sources:
+        - type: archive
+          url: https://files.pythonhosted.org/packages/16/8c/7a20a487f0f0cc83f718d696ad85149fa5164afd54c69d4a39eb01088540/Kivy%20Garden-0.1.4.tar.gz
+          sha256: 9b7d9de5efacbcd0c4b3dd873b30622a86093c9965aa47b523c7a32f3eb34610
+
+    # Runtime dependencies of Kivy
+    - name: libXmu
+      buildsystem: autotools
+      sources:
+      - type: archive
+        url: https://github.com/freedesktop/libXmu/archive/libXmu-1.1.2.tar.gz
+        sha256: fb1485a0ffcc360c0cc8fe18f54a2f49af6fc384da743c0b136ab99cc6c6e54c
+
+    - name: xclip
+      buildsystem: autotools
+      sources:
+        - type: archive
+          url: https://github.com/astrand/xclip/archive/0.13.tar.gz
+          sha256: ca5b8804e3c910a66423a882d79bf3c9450b875ac8528791fb60ec9de667f758
+
+    - name: mtdev
+      buildsystem: autotools
+      sources:
+        - type: archive
```

Comment:
to fix the aarch64 build here try and add something like https://github.com/flathub/com.tux4kids.tuxmath/blob/adc11b752a572c744b2041ba43d65545af121207/com.tux4kids.tuxmath.json#L51

Reply:
It's really painfully not working, even still. I'm actively considering commenting that module out until somebody complains about it. Would you believe I've been trying to package this thing for FIVE years?

Reply:
No! Wow, that is persistence. I can try and take a closer look a little later.

Reply:
Nah, I just dropped aarch64 support. I'd rather get *this* published and address aarch64 later. I just want this shipped.